### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by chrisjones120**
Unpin rack and move to Sinatra 2.

This resolves [CVE-2020-8130](https://github.com/advisories/GHSA-jppv-gw3r-w3q8), as Sinatra 1 was locked to an older version of Rake.

There were a couple of breaking changes such as the introduction of [Mustermann](https://github.com/sinatra/mustermann), which blocks anchors in regex, and the format of the cookie string timezone having changed.

### CODE REVIEW

File: spec/lib/discovery_service/application_spec.rb [hunk 1]
```diff
@@ -272,7 +272,7 @@ def run
 
     let(:reset_cookie) do
       'selected_organisations=; path=/; max-age=0; '\
-      'expires=Thu, 01 Jan 1970 00:00:00 -0000'
+      'expires=Thu, 01 Jan 1970 00:00:00 GMT'
```
Comment (bradleybeddoes) [inline spec/lib/discovery_service/application_spec.rb]: Should these be `UTC`?

Comment (smangelsdorf) [reply spec/lib/discovery_service/application_spec.rb]: Bump. @chrisjones120

Comment (chrisjones120) [reply spec/lib/discovery_service/application_spec.rb]: `GMT` is compliant with the HTTP/1.1 RFC [1][2], and this remains unchanged for HTTP/2 [3].

[1] - https://tools.ietf.org/html/rfc2616#section-3.3.1
>All HTTP date/time stamps MUST be represented in Greenwich Mean Time
   (GMT), without exception. For the purposes of HTTP, GMT is exactly
   equal to UTC (Coordinated Universal Time).

[2] - https://tools.ietf.org/html/rfc7234#section-5.3
[3] - https://tools.ietf.org/html/rfc7540#section-8

**Pull request merged**